packages/beacon-node/src/api/impl/beacon/pool/index.ts

```diff
@@ -163,13 +172,14 @@ export function getBeaconPoolApi({
 
     async submitPoolAttesterSlashings({attesterSlashing}) {
       await validateApiAttesterSlashing(chain, attesterSlashing);
-      chain.opPool.insertAttesterSlashing(attesterSlashing);
+      chain.opPool.insertAttesterSlashing(ForkName.phase0, attesterSlashing);
       await network.publishAttesterSlashing(attesterSlashing);
     },
 
     async submitPoolAttesterSlashingsV2({attesterSlashing}) {
-      // TODO Electra: Refactor submitPoolAttesterSlashings and submitPoolAttesterSlashingsV2
-      await this.submitPoolAttesterSlashings({attesterSlashing});
+      await validateApiAttesterSlashing(chain, attesterSlashing);
+      chain.opPool.insertAttesterSlashing(ForkName.electra, attesterSlashing);
```

Comment:
Hard coding to electra here seems fine since the api already ensures that correct `attesting_indices` length for pre-electra attester slashings